packages/blade/src/components/Collapsible/CollapsibleBody.tsx

```diff
@@ -1,20 +1,28 @@
 import type { ReactElement, ReactNode } from 'react';
 import { CollapsibleBodyContent } from './CollapsibleBodyContent';
 import { useCollapsible } from './CollapsibleContext';
+import type { BaseBoxProps } from '~components/Box/BaseBox';
 import BaseBox from '~components/Box/BaseBox';
 import { MetaConstants, assignWithoutSideEffects, makeAccessible, metaAttribute } from '~utils';
 import type { TestID } from '~src/_helpers/types';
 
 type CollapsibleBodyProps = {
   children: ReactNode;
+
+  /**
+   * **Internal**: used for React Native specific workarounds. Collapsible automatically takes care of width, you shouldn't need to configure this
+   */
+  _width?: BaseBoxProps['width'];
 } & TestID;
 
-const _CollapsibleBody = ({ children, testID }: CollapsibleBodyProps): ReactElement => {
-  const { collapsibleBodyId } = useCollapsible();
+const _CollapsibleBody = ({ children, testID, _width }: CollapsibleBodyProps): ReactElement => {
+  const { collapsibleBodyId, isExpanded } = useCollapsible();
   return (
     <BaseBox
       id={collapsibleBodyId}
-      {...makeAccessible({ role: 'region' })}
+      // Just React Native things, need this 100% so collapsed content flows correctly inside Accordion
+      width={_width}
```

Comment:
Instead of taking it from the consumer - which is sometimes the actual consumer and sometimes AccordionItem, can we not add a check for react native here directly and make it 100%? Otherwise this is a risky prop to expose to our consumers
```suggestion
      width={isReactNative ? '100%': undefined}
```

Reply:
We need `_width` regardless:
- for RN passing 100% width
- for Accordion to override Collapsible's internal responsive max / min width restrictions